src/services/measure/measure.js

```diff
@@ -0,0 +1,278 @@
+goog.provide('ngeo.interaction');
+goog.provide('ngeo.interaction.Measure');
+
+goog.require('goog.dom');
+goog.require('goog.dom.classlist');
+goog.require('goog.events');
+goog.require('ol.DrawEvent');
+goog.require('ol.Feature');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.MapBrowserEvent');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.interaction.Pointer');
+goog.require('ol.style.Style');
+
+
+
+/**
+ * @classdesc
+ * Interaction that allows measuring (length, area, ...).
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ * @param {{startMsg: (string|undefined),
+ *          style:(ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)}
+ *         =} opt_options Options
```

Comment:
Using a `@typedef` would be cleaner. (But this `@typedef` need no go in `ngeox.js` as this class is just a base class that won't be used directly in user code.)